pkg/controller/cluster_controller.go

```diff
@@ -58,9 +58,10 @@ type ClusterReconciler struct {
 // +kubebuilder:rbac:groups=core,resources=configmaps,verbs=get
 // +kubebuilder:rbac:groups=core,resources=configmaps/status,verbs=get
 // +kubebuilder:rbac:groups=core,resources=secrets,verbs=get;list;watch;create;update;patch
+// +kubebuilder:rbac:groups=core,resources=pods,verbs=get;list
```

Comment:
Did you take a look at all of the RBAC rules? We probably should.

Reply:
I started by removing everything that was in operator.yaml.in and then adding back the minimal set needed to get the operator to run correctly. I didn't try removing any of the existing markers. I think we should, but I'll tackle that in a separate PR.